src/training/data.py

```diff
@@ -357,17 +357,25 @@ def get_data(args, preprocess_fns, epoch=0):
 
     if args.train_data:
         data["train"] = get_wds_dataset(
-            args, preprocess_train, is_train=True, epoch=epoch)
+            args, preprocess_train, is_train=True, epoch=epoch, floor=False)
     if args.val_data:
         data["val"] = get_wds_dataset(
-            args, preprocess_val, is_train=False)
+            args, preprocess_val, is_train=False, floor=False)
     if args.image_data:
-        BUFFER = 0
+        np.random.seed(args.seed + epoch) # reseed every epoch
+        start_loc = np.random.randint(0, 1e7)
```

Comment:
why 1e7, this seems very arbitrary

Reply:
yea it is arbitrary. what do u suggest i replace 1e7 with? im thinking maybe an arg or something

Reply:
something like args.img_train_num_samples - samples_per_worker